src/SFA.DAS.LevyTransferMatching.Web/Orchestrators/PledgeOrchestrator.cs

```diff
@@ -220,6 +225,82 @@ public async Task<LocationSelectViewModel> GetLocationSelectViewModel(LocationSe
             };
         }
 
+        public async Task<byte[]> GetPledgeApplicationsDownloadModel(ApplicationsRequest request)
+        {
+            var result = await _pledgeService.GetApplications(request.AccountId, request.PledgeId);
+
+            var pledgeAppModel = new PledgeApplicationsDownloadModel
+            {
+                Applications = result.Applications?.Select(app =>
+                {
+                    var affordability = GetAffordabilityViewModel(app.Amount, app.PledgeRemainingAmount,
```

Comment:
As previous comment, I don't think it is needed (or elegant) to instantiate a vm intended for use with a partial that we don't use, only to get a single field out of it. I think the Estimate Cost This Year is in the `Amount` property of the application.